jdi-light/src/test/java/io/github/epam/properties/BrowserKillTests.java

```diff
@@ -0,0 +1,45 @@
+package io.github.epam.properties;
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.security.InvalidParameterException;
+import java.util.Properties;
+
+import static com.epam.jdi.light.common.Property.KILL_BROWSER;
+import static com.epam.jdi.light.common.PropertyValidationUtils.validateProperties;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.fail;
+
+public class BrowserKillTests {
+
+    @DataProvider
+    public static Object[] negativeData() {
+        return new Object[]{"avter", "afterAndBe", "be4"};
```

Comment:
Lets also check that property is not an empty string like "" or contains only digits "123"

Reply:
Done